README.md

```diff
@@ -33,14 +28,31 @@ In case of error try
 service netfilter-persistent save  
 service netfilter-persistent reload
 ```
+Now, install dependencies using `glide`.
+```
+curl https://glide.sh/get | sh
+glide install
+glide update
+```
+Download and setup glutton,
+```
+go get github.com/mushorg/glutton
+cd $GOPATH/src/github.com/mushorg/glutton
+go install app/server.go
+mkdir /etc/glutton
+cp $GOPATH/src/github.com/mushorg/glutton/rules/rules.yaml /etc/glutton
```

Comment:
When I was trying to install, I got an error that this file wasn't found in /etc/glutton